server/src/sequence.ts

```diff
@@ -35,7 +42,22 @@ export class MySequence implements SequenceHandler {
       const route = this.findRoute(request);
       const args = await this.parseParams(request, route);
       request.body = args[args.length - 1];
-      const authUser: User = await this.authenticateRequest(request);
+      const authUser= await this.authenticateRequest(request);
+      
+      if (authUser) {
+      //   console.log(`Please Login`);
+      // } else {
+        let role = await this.roleRepository.findById(authUser.rolekey);
+
+        const isAccessAllowed: boolean = await this.checkAuthorisation(
+          role.permissions,
```

Comment:
check if permissions are returned in authUser object.

Reply:
checked the authUser permission before checking authorisation